_posts/en/newsletters/2018-11-06-newsletter.md

```diff
@@ -0,0 +1,177 @@
+    To illustrate how easy it is to make this mistake, the example below
+    calls GBT with its default parameter and its segwit parameter---and
+    then compares the results by the total potential block reward
+    (subsidy + fees) each block template could earn.
+
+    ```bash
+    $ ## GBT with default parameters
+    $ bitcoin-cli getblocktemplate | jq '.coinbasevalue / 1e8'
+    12.54348709
+
+    $ ## GBT with segwit enabled
+    $ bitcoin-cli getblocktemplate '{"rules": ["segwit"]}' | jq '.coinbasevalue / 1e8'
+    12.56368175
+    ```
+
+    As you can see, a miner who enabled segwit would've earned more
+    income than a non-segwit miner if one of those example block
+    templates had been mined.  Although a small difference in absolute
+    terms  due to currently almost-empty mempools (about 0.02 BTC or
+    $100 USD), in relative terms the segwit-including example block
+    template receives almost 50% more fee income than the legacy-only
+    template.  This seems to be enough of an incentive to get miners to
+    create segwit-including blocks---and it will only become more
+    important in the future as more users switch to using segwit, the
+    block subsidy decreases, and perhaps fees increase.
+
+    [Bitcoin Core 0.17.0.1][] updated bitcoind's built-in documentation
+    for GBT to mention the need to enable segwit, and it has been proposed
+    in developer discussion to enable segwit GBT by default in some
+    future version (but still provide a backwards-compatible option to
+    disable it).
+
+- **Overflow bug in reference C-language bech32 implementation:** Trezor
+  [publicly disclosed][bech32 overflow blog] a bug they discovered in
+  the [reference implementation][bech32 c] of the bech32 address
+  function for the C programming language.  A [patch][bech32 patch] has
+  been released fixing the bug.  The bug does not affect users of the
+  other [reference implementations][bech32 refs] written in other
+  programming languages ([source][achow bech32]).
+
+    As Trezor responsibly disclosed the bug to multiple other projects,
+    they learned from Ledger about an additional bug in the
+    [trezor-crypto][] library for Bitcoin Cash-style addresses that use
+    the same basic structure as Bitcoin bech32 addresses.  A
+    [patch][cashaddr patch] for that has also been released.
+
+- **Discussion about improving Lightning payments:** in advance of an
+  upcoming meeting between LN protocol developers, Rusty Russel started
+  a [discussion][ln bolt11 ss] about two problems he thinks could
+  potentially be solved using scriptless scripts as described in
+  [Newsletter #16][].
+
+    1. An invoice can only be paid a maximum of one time.  It'd be nice
+       for multiple people to be able to pay the same invoice, such as a
+       static donation invoice or a monthly recurrent payment.
+
+    2. The protocol can't provide proof of payment.  You can prove that
```

Comment:
I think the first sentence here is confusing, since the next sentence seems to contradict it. How about replacing the first sentence with something like "It is not possible to prove that a particular party paid the invoice." or similar?